src/test/java/uk/gov/hmcts/divorce/citizen/notification/JointApplicationApprovedReminderTest.java

```diff
@@ -144,4 +145,33 @@ void shouldSendReminderEmailToApplicant1WithDissolutionAndNoPaymentContent() {
         );
         verify(commonContent).mainTemplateVars(data, 1234567890123456L, data.getApplicant1(), data.getApplicant2());
     }
+
+    @Test
+    void shouldSendReminderEmailInWelshToApplicant1WhenLanguagePreferenceIsWelsh() {
+        CaseData data = caseData();
+        data.setDueDate(LocalDate.now());
+        data.setDivorceOrDissolution(DivorceOrDissolution.DISSOLUTION);
+        data.setApplicant2(getApplicant(Gender.MALE));
+        data.getApplicant1().setLanguagePreferenceWelsh(YesOrNo.YES);
+        HelpWithFees hwf = HelpWithFees.builder().needHelp(YesOrNo.YES).build();
+        data.getApplication().setApplicant1HelpWithFees(hwf);
+        data.getApplication().setApplicant2HelpWithFees(hwf);
+
+        when(commonContent.mainTemplateVars(data, 1234567890123456L, data.getApplicant1(), data.getApplicant2()))
+            .thenReturn(getMainTemplateVars());
+
+        jointApplicationApprovedReminder.sendToApplicant1(data, 1234567890123456L);
+
+        verify(notificationService).sendEmail(
+            eq(TEST_USER_EMAIL),
+            eq(JOINT_APPLICATION_APPROVED_APPLICANT1_REMINDER),
+            argThat(allOf(
+                hasEntry(IS_REMINDER, YES),
+                hasEntry(PAYS_FEES, NO)
+            )),
```

Comment:
Can the Welsh content be verified?
hasEntry(PARTNER, "gŵr")

Reply:
done